packages/fccanalyses/package.py

```diff
@@ -24,8 +26,16 @@ class Fccanalyses(CMakePackage, Key4hepPackage):
     depends_on('edm4hep')
     depends_on('py-awkward')
```

Comment:
```suggestion
    depends_on('py-awkward@1.4.0')
```
There were some changes to the c++ interface after that, that would need fixes in FCCAnalyses